servicex_app/servicex/resources/transformation/submit.py

```diff
@@ -96,9 +97,20 @@ def post(self):
             args = self.parser.parse_args()
             config = current_app.config
 
-            image = args["image"]
+            image = args.get("image")
             did = args.get("did")
             file_list = args.get("file-list")
+            user_codegen_name = args.get("codegen")
+
+            code_gen_image_name = ''
+            for key, val in config['CODE_GEN_IMAGES'].items():
+                if key == user_codegen_name:
+                    code_gen_image_name = val
+
+            print("Before Code gen Image Name: ", code_gen_image_name)
```

Comment:
Get rid of development print statements.

Reply:
Removed Print statements